src/property.html

```diff
@@ -0,0 +1,1066 @@
+                    Budget
+                  </p>
+                  <div
+                    class="flex justify-between items-center gap-3 rounded-lg font-medium text-sm 2xl:text-lg leading-5 border-button px-5 py-4 2xl:px-5 2xl:py-6 bg-for-logo-section text-color-addition-info"
+                  >
+                    <p>Select Budget</p>
+                    <a href="#">
+                      <img
+                        src="/public/assets/propertypage/arrow-bottom.svg"
+                        alt="bottom"
+                      />
+                    </a>
+                  </div>
+                </div>
+                <!-- tenth -->
+                <div class="grid gap-2.5 sm:gap-3.5 2xl:gap-4 col-span-2">
+                  <p class="text-color 2xl:text-xl text-base font-semibold">
+                    Preferred Contact Method
+                  </p>
+                  <!-- div -->
+                  <div
+                    class="grid grid-flow-row sm:grid-cols-2 col-span-2 gap-4"
+                  >
+                    <div
+                      class="flex justify-between gap-1.5 2xl:gap-3 items-center rounded-lg col-span-1 font-medium text-sm 2xl:text-lg leading-5 border-button px-5 py-4 2xl:px-5 2xl:py-6 bg-for-logo-section text-color-addition-info"
+                    >
+                      <div class="flex items-center gap-1.5 2xl:gap-3">
+                        <img
+                          src="/public/assets/propertypage/telephone.svg"
+                          alt="telephone-icon"
+                        />
+                        <p>Enter Your Name</p>
+                      </div>
+                      <img
+                        src="/public/assets/propertypage/circle-purple.svg"
+                        alt="bg-purple"
+                      />
+                    </div>
+                    <div
+                      class="flex justify-between gap-1.5 2xl:gap-3 items-center rounded-lg col-span-1 font-medium text-sm 2xl:text-lg leading-5 border-button px-5 py-4 2xl:px-5 2xl:py-6 bg-for-logo-section text-color-addition-info"
+                    >
+                      <div class="flex items-center gap-1.5 2xl:gap-3">
+                        <img
+                          src="/public/assets/propertypage/email-icon.svg"
+                          alt="email"
+                        />
+                        <p>Enter Your Email</p>
+                      </div>
+                      <div class="p-2 border-star rounded-full"></div>
+                    </div>
+                  </div>
+                </div>
+                <!-- last one eleven -->
+                <div
+                  class="flex flex-col col-span-2 lg:col-span-4 gap-2.5 lg:gap-3.5 2xl:gap-4"
+                >
+                  <p class="font-semibold text-base 2xl:text-xl text-color">
+                    Message
+                  </p>
+                  <p
```

Comment:
fuck, brother, how a user can add text to a paragraph? are you in your fucking senses??

Reply:
sorry,it it make it for ui i really fogot to change it

Reply:
![image](https://github.com/user-attachments/assets/4171be57-b4ed-4452-99b3-60e0b4de85e0)
fully inputs